libs/api-client/src/api-client.ts

```diff
@@ -153,10 +161,15 @@ export class GFW_API_CLASS {
     }
   }
 
+  setDefaultApiVersion(version: ApiVersion) {
+    this.apiVersion = version
+  }
+
   setConfig(config: LibConfig) {
-    const { debug = this.debug, baseUrl = this.baseUrl, dataset = this.dataset } = config
+    const { debug = this.debug, baseUrl = this.baseUrl, dataset = this.dataset, version = this.apiVersion } = config
     this.debug = debug
     this.baseUrl = baseUrl
+    this.apiVersion = version
     this.dataset = dataset
```

Comment:
could we get rid of the dataset property? I think it is not used anymore so we can also simplify the url generation [here](https://github.com/GlobalFishingWatch/frontend/blob/migrate-new-api-client/libs/api-client/src/api-client.ts#L319)